### PR TITLE
Add ability to install man pages

### PR DESCRIPTION
**Pull request by rhatdan**
We want to create a containers-image package so that container-common can
include all of these man pages in the distribution.

Replaces: https://github.com/containers/skopeo/pull/645

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@lsm5 @mtrmac @vrothberg PTAL

File: Makefile [hunk 1]
```diff
@@ -31,6 +37,17 @@ build: vendor build-internal
 build-internal:
 	$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
 
+$(MANPAGES): %: %.md
+	@sed -e 's/\((.*\.5.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' $<  | $(GOMD2MAN) -in /dev/stdin -out $@
```
Comment (mtrmac) [inline Makefile]: I can’t see that the two sed commands match anything in existing man pages. The skopeo reference is especially suspect.

File: Makefile [hunk 2]
```diff
@@ -56,7 +73,7 @@ vendor: tools.timestamp vendor.conf
 	@touch vendor
 
 clean:
-	rm -rf vendor tools.timestamp
+	rm -rf vendor tools.timestamp docs/*.5
```
Comment (mtrmac) [inline Makefile]: `$(MANPAGES)` please, to minimize duplication.

**Comment by lsm5** [comment]
> ACK overall, but this needs to also be ACKed by @lsm5 , because new RPMs will be necessary. (Or is that Jindřich now?)

Jindřich for RHEL. Me for everything else :) . I'll check this out and get back in a bit.

File: Makefile (hunk 1 above)
Comment (rhatdan) [reply Makefile]: Yup cut & paste from skopeo Makefile (fixed)

File: Makefile (hunk 2 above)
Comment (rhatdan) [reply Makefile]: Fixed

**Comment by rhatdan** [comment]
I am going to merge this and @lsm5 and @jnovy can work on packaging up container-storage and containers-image separately.

**Pull request merged**